demo_data/floating_lidar_demo_iea43_wra_data_model.json

```diff
@@ -54,6 +54,16 @@
         "date_from": "2022-03-01T12:10:00",
         "date_to": null,
         "update_at": "2022-03-31T12:13:27"
+       },{
+	"logger_stated_device_datum_plane_height": 1.5,
```

Comment:
Would you not just add `logger_stated_device_datum_plane_height` to the existing config? The way you have it you have two lidar configs for the same point in time. One for a flow correction and the other for this.